psi4/driver/driver.py

```diff
@@ -82,6 +82,9 @@ def _find_derivative_type(ptype, method_name, user_dertype):
             raise ValidationError("_find_derivative_type: user_dertype should only be None or int!")
         dertype = user_dertype
 
+    if (core.get_global_option('INTEGRAL_PACKAGE') == 'ERD') and (dertype != 0):
+        raise ValidationError("INTEGRAL_PACKAGE ERD does not play nicely with gradients, so stopping.")
+
```

Comment:
Probably need one in Hessian too since we have real Hessians as well. For DFT we can probably stick one in the super functional constructor that checks `sup.is_x_lrc() or sup.is_c_lrc()`. I added one to the DFT part of the 1.2 branch.